docs/audits/total-ad-blocking-time.md

```diff
@@ -0,0 +1,18 @@
+# Reduce Ad JS Blocking Time
+
+## Overview
+
+This audit measures how much [blocking time](https://web.dev/tbt/) is
+attributable to ads scripts. This captures time where the JS thread is blocked
+by an ad script for long enough to be noticeable by users when it impacts
+input delay or content loading.
+
+## Recommendations
+
+The most important factor in ad JS blocking time is how many ads are being
+loaded. Our recommendations are:
```

Comment:
Isn't it more *when* ads are being loaded, rather than the overall number? If so, maybe reword to "...how many ads are being loaded at once" or something.

Reply:
Done. Although I imagine there is some correlation with the total number of ads even if they're split into small batches

Reply:
I don't see any change here, but I'm fine leaving as-is if you think the number of ads is actually the most important factor.